src/components/Profile/ProfileInfo/ProfileStatus.tsx

```diff
@@ -0,0 +1,38 @@
+import { createSelector } from 'reselect';
+import { AppStateType } from '../../../redux/redux-store';
+
+export const getUsersSelector = (state: AppStateType) => {
+  return state.usersPage.users;
+}
+
+export const getUsers = createSelector(getUsersSelector, (users) => {
+  return users.filter(u => true);
+})
+
+// export const getUsers = (state) => {
+//   return state.usersPage.users;
+// }
+
+// export const getUsersSuperSelector = createSelector(getUsers, getIsFetching, (users, isFetching) => {
+//   return users.filter(u => true);
+// })
```

Comment:
remove

Reply:
For educational purposes I will leave this code